app/src/main/java/com/safetyheads/akademiaandroida/views/BackArrowView.java

```diff
@@ -0,0 +1,41 @@
+package com.safetyheads.akademiaandroida.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import androidx.annotation.DrawableRes;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.safetyheads.akademiaandroida.R;
+
+public class BackArrowView extends FrameLayout {
```

Comment:
I think you can just extend ImageView :)

Reply:
That's not a good idea I think, since we try to make a Toolbar kind of view here, right? @msarelo